lib/template/config/config.rb

```diff
@@ -1,5 +1,9 @@
 require "pliny/config_helpers"
 
+
+# Supress the "use RbConfig instead" warning.
+Object.send(:remove_const, :Config) if Object.const_defined?(:Config)
```

Comment:
Would it be possible to leave this in the config helper, but add the module existence check to the conditional? I'd prefer to keep the templated config smooth.